### DISCUSSION TITLE
UNO Platform App build errors on Windows only.

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#13323] [Q&A] [answered]
Hi,

I am trying to build a project (UNO Platform App) that only Targets windows

The options I am slecting are:

Framework: .NET 7
Platforms: Windows and WPF
Presentation: MVUX
Theme: Fluent
Extensions: None Selected
Projects: Server not selected
Features: Non Selected
Authentication: None
Application: Default Values
Testing: Non Selected

In my project file: UnoTest1.Windows.csproj

I make the following change for Unpackaged builds

	<PropertyGroup>
		<OutputType>WinExe</OutputType>
		<TargetFramework>net7.0-windows10.0.19041.0</TargetFramework>
		<TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
		<RootNamespace>UnoTest1.Windows</RootNamespace>
		<ApplicationManifest>app.manifest</ApplicationManifest>
		<Platforms>x86;x64;arm64</Platforms>
		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
		<PublishProfile>win10-$(Platform).pubxml</PublishProfile>
		<UseWinUI>true</UseWinUI>
		<EnableMsixTooling>true</EnableMsixTooling>
		
		<WindowsPackageType>None</WindowsPackageType>
    	
	</PropertyGroup>

The code builds and runs however when I try to look at the file MainPage.xaml

I get the following errors:

<Page x:Class="UnoTest1.MainPage"
	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
	xmlns:local="using:UnoTest1"
	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
	mc:Ignorable="d"
	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">

	<StackPanel HorizontalAlignment="Center"
				VerticalAlignment="Center">
		<TextBlock AutomationProperties.AutomationId="HelloTextBlock"
				   Text="Hello Uno Platform" />
	</StackPanel>
</Page>

Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'Page' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	UnoTest1	E:\Data\Users\Ward\Documents\Visual_Studio_Projects\UnoTest1\UnoTest1\UnoTest1\MainPage.xaml	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'StackPanel' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	UnoTest1	E:\Data\Users\Ward\Documents\Visual_Studio_Projects\UnoTest1\UnoTest1\UnoTest1\MainPage.xaml	10	

Severity	Code	Description	Project	File	Line	Suppression State
Error	XLS0414	The type 'TextBlock' was not found. Verify that you are not missing an assembly reference and that all referenced assemblies have been built.	UnoTest1	E:\Data\Users\Ward\Documents\Visual_Studio_Projects\UnoTest1\UnoTest1\UnoTest1\MainPage.xaml	12	

My questions are:

1. How can I resolve this?
2. Is thre anything extra in my defaults like some dependancies that are required for me to select
   to make this work when going through the wizard.

Thanks
Ward

![Screenshot 2023-08-21 142108](https://github.com/unoplatform/uno/assets/12206371/5efe74c9-1fda-47be-bb27-70bb0749d6d2)
![Screenshot 2023-08-21 142210](https://github.com/unoplatform/uno/assets/12206371/f51c49b3-6d3d-4b30-a399-86632c05253b)
![Screenshot 2023-08-21 142259](https://github.com/unoplatform/uno/assets/12206371/8773d812-4fcf-46f8-9321-e93fa346fe10)

**Comment by jeromelaban** [comment]
Thanks for the report! Those are not build errors, but rather intellisense errors that Visual Studio is reporting incorrectly and does not affect the ability to build the app.

You can find a [resolution here](https://platform.uno/docs/articles/get-started-wizard.html?tabs=windows#the-xaml-editor-shows-the-type-page-does-not-support-direct-content-message).

**Reply by ward-horsfall** [reply]
Hi jeromelaban, 

Thanks for your response I will test out your answer.  Should I raise it as an issue so it gets resolved?

Thanks
Ward

**Reply by jeromelaban** [reply, marked as answer]
Thanks for the update. We already have opened issues about this topic, and this something that is managed by Microsoft and we're working with them to get this addressed.
